scripts/no-secrets.sh

```
@@ -1,60 +1,109 @@
+    if [ -n "$CONTENT_LEAK" ]; then
+        CONTENT_FOUND=true
+    fi
+fi
+
+if [ "$CONTENT_FOUND" = true ]; then
+    echo "BLOCKED: API key pattern detected; matched values were redacted."
+    if [ "$SCOPE" = "tracked" ]; then
+        MATCH_COUNT=0
+        while IFS= read -r -d '' file \
+            && IFS= read -r -d '' line \
+            && IFS= read -r content; do
```

Shellcheck (0.11.0)

[warning] content appears unused. Verify use (or export if used externally).

(SC2034)